packages/ui/src/UseAutocomplete.jsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from "react";
```

Comment:
The components needed to have better naming. 
- `use` prenom is only for hooks, in this case, is used in an Autocomplete component.
- `ChildComponents` is used only for Search related components, in order to keep order it needs to be inside a Seach folder that wraps all the components related to the search
- in `packages/src/utils` naming with a capital first letter is only for components, in helpers use camelCase